README.md

```diff
@@ -18,14 +18,19 @@ import { ReadmeBox } from 'readme-box'
 You can quickly update a section of a README:
 
 ```js
-await ReadmeBox.updateSection('New contents!', {
+const result = await ReadmeBox.updateSection('New contents!', {
   owner: 'JasonEtco',
   repo: 'example',
   token: process.env.GITHUB_TOKEN,
   // branch is assumed to be 'master' by default, you can also specify `branch: 'main'`
   branch: 'main',
-  section: 'example-section'
+  section: 'example-section',
+  // set to false to avoid empty commits in case of no changes
+  emptyCommits: true
 })
+
+// result is the response object of the README update request or
+// undefined in case there were no changes and emptyCommits is set to false
```

Comment:
note this change in behavior if there was no update commit.

I wonder it would to change the result. You could e.g. always return an object with a `{ changed }` key that is always set. And if `changed` is true, then you could add a `response` key?

An alternative would be to return the response object of the `GET` request if there was no commit, but that would require a change to `.getReadme()`

Reply:
Thanks for calling this out - the return value of `updateSection` shouldn't be important since its a `write` operation with all of the same info you'd get in the response already available (maybe minus the new commit SHA).

So, I'm not worried about this change, and I think returning `undefined` is fine. Users would just check `if (updateResponse)` or something.